drizzlepac/haputils/cell_utils.py

```diff
@@ -300,6 +303,25 @@ def build(self, expnames, scale=False, scale_kw='EXPTIME'):
 
             self.total_mask += self.exp_masks[exposure]['mask']
 
+            # Compute the bounded WCS for this footprint
+            self.find_bounded_wcs()
+
+
+    def find_bounded_wcs(self):
+        """Compute the WCS based on the bounding box of footprint """
```

Comment:
This is a terminology comment. In GWCS/astropy and hence JWST and Roman, the bounding box is defined in the inputs space, i.e. it's in pixels, while the footprint is defined in outputs space, i.e. on sky.
So essentially bounding box and footprint are the same thing but in different coordinate frames.
Unless there's a good reason it's best to keep similar terminology to decrease confusion.

Reply:
The comment here and earlier has been edited.  The use of the term 'bounding box' seems to be consistent with what is used elsewhere since it is specified here in pixel space, but footprint was misused, so this reference  to 'footprint' was reworded, along with comment associated with the call to this function.  The changes are in 6abb063.  Hopefully, this is sufficient at this time.